qsimcirq/qsim_simulator.py

```diff
@@ -16,18 +16,7 @@
 from dataclasses import dataclass
 from typing import Any, Dict, List, Optional, Sequence, Tuple, Union
 
-from cirq import (
-    circuits,
-    devices,
-    ops,
-    protocols,
-    sim,
-    study,
-    value,
-    SimulatesAmplitudes,
-    SimulatesFinalState,
-    SimulatesSamples,
-)
+import cirq
 
 # TODO: import from cirq directly when fix is released
 from cirq.sim.simulator import SimulatesExpectationValues
```

Comment:
This TODO is resolved now - `cirq.SimulatesExpectationValues` can replace this.

Reply:
Done.